evansmusomi/react-native-102
# Installs and Uses React Native Image Picker

*What this does*
- Installs React Native Image Picker
- Uses Image Picker to Select Place Image on Share Place screen

*How to verify*
- Navigate to the Share Place screen
- Press the "Pick Image" button
- You should be prompted to pick an image from your gallery or take a photo
- The image you select should appear on the Share Screen Image Preview
